applications/bwa_alignment/values.yaml

```diff
@@ -0,0 +1,109 @@
+values:
+  - name: ReferenceGenomeURL
+    display_name: Reference Genome URL
+    uint_value:
```

Comment:
i think you should remove the uint_value key for all value entries that are strings (and vice versa)